DigitalOffice/Services/ProjectService/Validators/EditProjectValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Models;
+
+namespace LT.DigitalOffice.ProjectService.Validators
+{
+    public class EditProjectValidator : AbstractValidator<EditProjectRequest>
+    {
+        public EditProjectValidator()
+        {
+            RuleFor(project => project.Name)
+                    .NotEmpty()
```

Comment:
Is the name of the project always required to indicate when making changes? Or is the identifier enough?

Reply:
Added Id validation.
For the name...I think it should come either way from the front-end: changes or not.
So that .Update() method wouldn't erase the project name.

So yea, project name is required.